content/financial-aid.html

```diff
@@ -57,9 +65,9 @@
 <h2>Applying</h2>
 
 <p>
-    This year we are running our financial aid process alongside our ticket
-    sales, using the same service. This allows us to centralise our data
-    collection.
+    To apply for financial aid, please complete the form at the top of this
+    page.  We'll be in contact when we've decided the status of your grant
+    application.  The deadline for financial aid applications is June 28th.
```

Comment:
Why not link to the form here, as well as at the top of the page, to save the reader that extra bit of scrolling?